frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -481,10 +481,26 @@ export const experimentLogic = kea<experimentLogicType>([
             values.experiment && actions.reportExperimentArchived(values.experiment)
         },
         updateExperimentGoal: async ({ filters }) => {
-            // We never want to update global properties in the experiment
+            const { recommendedRunningTime, recommendedSampleSize, minimumDetectableChange } = values
```

Comment:
would it be better here to give the insight time to load first? So, for example, disable the save button until the insight loads / errors out. Feels a bit too opaque otherwise imo

Reply:
yes, but for that we need to pull the loading state from dataNodeLogic - not sure if it's worth complicating things here.

Anyway, if we do want to do it, what would be the correct props to pass to dataNodeLogic?

This is how dataNodeLogic is connected to insightDataLogic:
```
dataNodeLogic({
    key: insightVizDataNodeKey(props),
    loadPriority: props.loadPriority,
} as DataNodeLogicProps),
[
    'responseLoading',
],
```

Reply:
actually this seems to be all we need? -> https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/insights/insightDataLogic.tsx#L61 and its already connected

Reply:
perfect 👌